client/me/concierge/pending-appointment/pending-appointment.js

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { localize, moment } from 'i18n-calypso';
+import getConciergeNextAppointment from 'state/selectors/get-concierge-next-appointment';
+import Confirmation from '../shared/confirmation';
+import Button from 'components/button';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class PendingAppointment extends Component {
+	static propTypes = {
+		nextAppointment: PropTypes.object.isRequired,
+		site: PropTypes.object.isRequired,
+	};
+
+	componentDidMount() {
+		this.props.recordTracksEvent( 'calypso_concierge_pending_appointment' );
+	}
+
+	render() {
+		const { translate } = this.props;
+		const beginTimestamp = moment( this.props.nextAppointment.beginTimestamp ).format( 'LLLL' );
```

Comment:
I booked a session at 12:00pm Eastern time, but it's displaying as 11:00am because my browser is in Central time. Make sure to convert to the appointment-specific timezone and actually display the timezone (e.g. "12:00pm EDT")